extensions/runtimes/src/index.ts

```diff
@@ -129,7 +129,9 @@ export default async (composer: any): Promise<void> => {
         await copyDir(schemaSrcPath, localDisk, schemaDstPath, project.fileStorage, pathsToExclude);
         const schemaFolderInRuntime = path.join(destPath, 'azurewebapp/Schemas');
         await removeDirAndFiles(schemaFolderInRuntime);
-        return destPath;
+        console.log(project.dir);
```

Comment:
please remove the debugging message here

Reply:
done